packages/prisma/zod-utils.ts

```diff
@@ -19,7 +19,7 @@ export const eventTypeLocations = z.array(
   z.object({
     type: z.nativeEnum(LocationType),
     address: z.string().optional(),
-    link: z.string().url().optional(),
+    link: z.string().optional(), // TODO: Refine this validation based on type - with this approach we are only validating on the FE
```

Comment:
Why `url` was removed?

Reply:
As link isn't always in a ULR format - We can't validate for the Fake app links and also use .url

Reply:
Why not? A link should be a valid URL unless I'm missing something

Reply:
I was under the impression we should be able to pass in "www.XYZ.com" - zods URL fails this as a valid URL. We could refine the type here and make our own type to test URLs - to make sure these are always URLs `https://` `https://www.` `www.`

Reply:
Nope, that's a can of worms we don't want to open. Only valid URLs for now.

Reply:
agree with @zomars. never write your own URL validators ☝️